database/src/main/postgres/runs/V1.5.11__get_partitioning_measures.sql

```diff
@@ -33,29 +33,48 @@ $$
 -- Returns:
 --      measure_name        - Name of the measure
 --      measured_columns    - Array of columns associated with the measure
+--      status              - Status code
+--      status_text         - Status message
+
+-- Status codes:
+--      10 - Record not found for the given partitioning
```

Comment:
**Minor:**
I would make this 16, to avoid the numbers we have [convention for](https://github.com/AbsaOSS/fa-db/blob/master/core/src/main/scala/za/co/absa/fadb/status/README.md).